packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -340,7 +340,7 @@ export class ModelScene extends Scene {
         return;
       }
       const material = mesh.material as Material;
-      if (!material.transparent) {
+      if (!(material as any).isMeshBasicMaterial || !material.transparent) {
```

Comment:
We made this change for a reason; we found too many models with baked shadow planes that were not using an unlit shadow material. I believe there's another way around this.